website/docs/webdev/3-Guidelines/1-typography.mdx

```diff
@@ -78,63 +75,63 @@ We set 16px (1rem) as the base font size for body text to ensure readability. It
             <h1 className="h1">Heading 1</h1>
         </div>
         <div className="col col--6">
-            <b>Font Size: 44x or 2.75rem  <br/> Letter spacing: -0.015em <br/> Line height: 1.2</b>
+            <b>Font Size: 44x or 2.75rem  <br/> Letter spacing: -1.5% <br/> Line height: 120%</b>
```

Comment:
@g-stamatis  I've written in the current doc why I reference -1.5% as a figma value but I show -0.015em since in CSS code that is how you turn -1.5% into a value that can be used by letter spacing